sled-agent/src/services.rs

```diff
@@ -3712,17 +3733,8 @@ impl ServiceManager {
         Ok(())
     }
 
+    /// Adjust the system boot time to the latest boot time of all zones.
     pub fn boottime_rewrite(&self) {
```

Comment:
Can / should we make this private, now that it should be called through the `if_timesynced()` method?